app/src/main/java/me/quadphase/qpdex/databaseAccess/PokemonFactory.java

```diff
@@ -296,27 +403,10 @@ public void getAllDetailedPokemon(){
 
             if(PRINT_DEBUG)
                 Log.d("QPDEX",String.format("Building %s",i));
-//                //We have to slow down the loop intentionally...
-//                try {
-//                    Thread.sleep(10);
-//                }
-//                catch(InterruptedException e){
-//
-//                }
         }
 
     }
 
-    //TODO: FIX thread classes before calling again.
-    public void getAllDetailedPokemonInParallel(){
-        Thread masterBuilder = new DetailedListMaster();
-        masterBuilder.start();
-        try {
-            masterBuilder.join();
-        } catch (InterruptedException e) {
-            e.printStackTrace();
-        }
-    }
 
     private void buildPartOfDetailedPokemonList(int start, int end){
         for (int i = start; i < end; i++) {
```

Comment:
I can't comment on lines that were unchanged, but do you think it's necessary to keep the getMinimalPokemonByNationalID(...) method at line 434? Is this used anywhere anymore (no) or will it ever be used? At the very least, it can be optimized to just get the MinimalPokemon from the already constructed list.

The next method, getMinimalPokemonByPokemonID(...), might still be used for the alt forms, so we might want to keep that one though.